packages/nx/src/project-graph/file-utils.ts

```diff
@@ -155,10 +154,3 @@ export function readPackageJson(): any {
 }
 // Original Exports
 export { FileData };
-
-// TODO(v16): Remove these exports
-export {
-  readNxJson,
-  readAllWorkspaceConfiguration,
-  workspaceLayout,
-} from '../config/configuration';
```

Comment:
Sorry, we actually have to keep these. `@nrwl/devkit@15.4.0` depends on them. https://unpkg.com/@nrwl/devkit@15.4.0/index.js

We'll have to keep these until 17. Could you please change the TODO?

Reply:
added back